nautobot/extras/models/jobs.py

```diff
@@ -859,7 +859,10 @@ class JobButton(BaseModel, ChangeLoggedModel, NotesMixin):
         choices=ButtonClassChoices,
         default=ButtonClassChoices.CLASS_DEFAULT,
     )
-    confirmation = models.BooleanField(help_text="Enable confirmation pop-up box", default=True)
+    confirmation = models.BooleanField(
+        help_text="Enable confirmation pop-up box. <font color='#ff0000'>WARNING: unselecting this option will allow the Job to run (and commit changes) with a single click!</font>",
```

Comment:
Can you do
```suggestion
        help_text="Enable confirmation pop-up box. <span class='text-danger'>WARNING: unselecting this option will allow the Job to run (and commit changes) with a single click!</span>",
```
instead?